content/docs/dvclive/api-reference/live/index.md

```diff
@@ -62,11 +62,14 @@ other metadata.
 
   </admon>
 
-- `report` - If `html`, DVCLive will call `Live.make_report()` on each step
-  update. _Default_: `html`.
+- `report` - If `auto`,`html` or `md`, DVCLive will call `Live.make_report()` on
+  each step update. _Default_: `auto`.
```

Comment:
```suggestion
- `report` - If `auto`,`html`, or `md`, DVCLive will call `Live.make_report()` on
  each step update. The `auto` mode (default) will use `md` format if a `CI` env
  var is present, otherwise it will use `html`.
```